incubator/etcd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: etcd
 home: https://github.com/coreos/etcd
-version: 0.3.9
+version: 0.3.10
```

Comment:
This should be a major bump. Once this is through, we should move the chart to stable.